WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -293,6 +295,19 @@ class PostCoordinator: NSObject {
         startSync(for: revision.original())
     }
 
+    func retrySync(for post: AbstractPost) {
+        wpAssert(post.isOriginal())
+
+        guard let revision = post.getLatestRevisionNeedingSync() else {
+            return
+        }
+        revision.confirmedChangesTimestamp = Date()
```

Comment:
This is the part where the app "restarts" the `SyncWorker.maximumRetryTimeInterval` timer.